src/components/NavBar/MenuDropdown.tsx

```diff
@@ -140,24 +146,37 @@ export const MenuDropdown = () => {
                 <Box display={{ sm: 'none', lg: 'flex', xxl: 'none' }}>
                   <PrimaryMenuRow to="/pool" close={toggleOpen}>
                     <Icon>
-                      <PoolIcon width={24} height={24} color={theme.textSecondary} />
+                      <PoolIcon width={24} height={24} fill={theme.textPrimary} />
                     </Icon>
                     <PrimaryMenuRow.Text>
                       <Trans>Pool</Trans>
                     </PrimaryMenuRow.Text>
                   </PrimaryMenuRow>
                 </Box>
+                <Box
+                  display={mgtmEnabled ? 'flex' : 'none'}
```

Comment:
Are we intentionally showing this for all devices, and not just for iOS devices? eg in the banner the language changes if !iOS to "Learn more" instead of "Download".

Reply:
Yes, `openDownloadApp` opens the microsite if not on iOS, which is the desired functionality.

Reply:
This feels inconsistent. Could you check in with design on this?

Reply:
They're in the loop on this functionality.

Reply:
+1 can confirm this is desired behavior